catalog/core/catalog-core-subscriptionstore/subscriptionstore-impl/pom.xml

```diff
@@ -42,6 +42,26 @@
             <artifactId>cache-api</artifactId>
             <version>1.0.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.cache2k</groupId>
+            <artifactId>cache2k-all</artifactId>
+            <version>1.0.0.CR4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.cache2k</groupId>
+            <artifactId>cache2k-jcache</artifactId>
+            <version>1.0.0.CR4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.cache2k</groupId>
+            <artifactId>cache2k-api</artifactId>
+            <version>1.0.0.CR4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.cache2k</groupId>
+            <artifactId>cache2k-jcache-api</artifactId>
+            <version>1.0.0.CR4</version>
+        </dependency>
```

Comment:
I will move the above versions to a maven property. I'm guessing that property should stay in a local subscription store pom. Agree?